packages/block-library/src/code/index.js

```diff
@@ -20,7 +16,7 @@ export const settings = {
 
 	description: __( 'Add text that respects your spacing and tabs -- perfect for displaying code.' ),
 
-	icon: <SVG viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><Path d="M0,0h24v24H0V0z" fill="none" /><Path d="M9.4,16.6L4.8,12l4.6-4.6L8,6l-6,6l6,6L9.4,16.6z M14.6,16.6l4.6-4.6l-4.6-4.6L16,6l6,6l-6,6L14.6,16.6z" /></SVG>,
+	icon: <svg viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M0,0h24v24H0V0z" fill="none" /><path d="M9.4,16.6L4.8,12l4.6-4.6L8,6l-6,6l6,6L9.4,16.6z M14.6,16.6l4.6-4.6l-4.6-4.6L16,6l6,6l-6,6L14.6,16.6z" /></svg>,
```

Comment:
Not sure about `path`, `g` and rest, but at least `SVG` needs to stay as it adds some accessibility features behind the scenes. It's expected to have it everywhere.

Reply:
Couldn't we just map `svg` to our `SVG` then (instead of the one from react-native-svg`)?